ads/index.js

```diff
@@ -73,6 +74,9 @@ function onAdsComplete (event) {
 			utils.log.info('Ad loaded in slot', event);
 			if (slotsRendered === 0) {
 				perfMark('firstAdLoaded');
+				if (!/spoor-id=0/.test(document.cookie)) {
```

Comment:
I don't think it really matters, since we can't tie the two events up nicely anyway